### PR TITLE
feat: seeding performance monitoring 

### PR DESCRIPTION
**Pull request by LuisFelipeCoelho**

### CODE REVIEW

File: CI/physmon/phys_perf_mon.sh [hunk 1]
```diff
@@ -91,8 +99,8 @@ run \
     $outdir/acts_analysis_residuals_and_pulls.root \
     $refdir/acts_analysis_residuals_and_pulls.root \
     --title "analysis_residuals_and_pulls" \
-#    -o $outdir/analysis_residuals_and_pulls.html \
-#    -p $outdir/analysis_residuals_and_pulls
+    -o $outdir/analysis_residuals_and_pulls.html \
+    -p $outdir/analysis_residuals_and_pulls
```
Comment (paulgessinger) [inline CI/physmon/phys_perf_mon.sh:103]: This was previously producing a **lot** of plots, and we turned it off again (cf @andiwand)

Comment (LuisFelipeCoelho) [reply CI/physmon/phys_perf_mon.sh:103]: sorry, I will remove that

**Comment by paulgessinger** [comment]
This looks pretty good already, thank!

**Review by andiwand — COMMENTED** [review]
great to have this! looking forward to the first plots

**Comment by LuisFelipeCoelho** [comment]
@paulgessinger should I also add the monitoring for the orthogonal seeding?

**Comment by paulgessinger** [comment]
If you can, that would of course be brilliant.

**Comment by LuisFelipeCoelho** [comment]
I don't know why the github-actions doesn't show the new plots

**Comment by paulgessinger** [comment]
> I don't know why the github-actions doesn't show the new plots

Ha sorry, that's not obvious: they only show up when the template is merged, i.e. it's updated on the main branch.

**Comment by LuisFelipeCoelho** [comment]
The IntegralTest of the residuals and pulls comparison for the orthogonal seeding is failing. Do you know why's this could be happening?
```
Error: all_pull_smt_t: IntegralTest
  Intregal: 24.017602920532227+-0.015849246649194293 vs. 23.913843154907227+-0.015776328269696355: (int_a - int_b) / sqrt(sigma(int_a)^2 + sigma(int_b)^2) = 4.64 >= 3
  🔴 IntegralTest Intregal: 24.017602920532227+-0.015849246649194293 vs. 
  23.913843154907227+-0.015776328269696355: (int_a - int_b) / sqrt(sigma(int_a)^2 
  + sigma(int_b)^2) = 4.64 >= 3
```

**Comment by andiwand** [comment]
I think it should be fine to run residuals and pulls with only one seeding algorithm. what do you think @paulgessinger ?

**Comment by paulgessinger** [comment]
Probably, yeah. Also I'm noticing that the performance monitoring says "tracking_eff_eta" etc, but that's the seeding efficiency, right?

**Comment by LuisFelipeCoelho** [comment]
yes, I agree that the name should be changed, but I think I will make another PR for it, because I also want to include more variables in the seeding performance

**Comment by paulgessinger** [comment]
Orthogonal seeding reference seems to be off. Should this already match @LuisFelipeCoelho ?

**Comment by LuisFelipeCoelho** [comment]
@paulgessinger I will update the plots

**Comment by LuisFelipeCoelho** [comment]
I hope now the physmon won't fail

**Review by paulgessinger — APPROVED** [review]

**Pull request merged**